src/main/java/com/exasol/adapter/dialects/oracle/OracleConnectionDefinitionBuilder.java

```diff
@@ -27,33 +22,15 @@ public String buildConnectionDefinition(final AdapterProperties properties,
     }
 
     private String buildImportFromOraConnectionDefinition(final AdapterProperties properties) {
-        if (hasConflictingConnectionProperties(properties)) {
-            throw new IllegalArgumentException(CONFLICTING_CONNECTION_DETAILS_ERROR);
-        } else if (properties.containsKey(ORACLE_CONNECTION_NAME_PROPERTY)
-                && hasIndividualConnectionPropertiesOnly(properties)) {
-            return mixConnectionPropertiesWithOracleConnectionName(properties);
-        } else if (properties.containsKey(ORACLE_CONNECTION_NAME_PROPERTY)) {
+        if (properties.containsKey(ORACLE_CONNECTION_NAME_PROPERTY)) {
             return buildOracleConnectionDefinitionFromOracleConnectionOnly(properties);
         } else {
-            throw new IllegalArgumentException("If you enable IMPORT FROM ORA with property " + ORACLE_IMPORT_PROPERTY
-                    + " you also need to provide the name of an Oracle connetion with "
-                    + ORACLE_CONNECTION_NAME_PROPERTY + " and credentials in " + USERNAME_PROPERTY + " and "
-                    + PASSWORD_PROPERTY + ".");
+            throw new IllegalArgumentException("If you enable IMPORT FROM ORA with property \"" + ORACLE_IMPORT_PROPERTY
+                    + "\" you also need to provide the name of an Oracle connection with \""
+                    + ORACLE_CONNECTION_NAME_PROPERTY + "\".");
```

Comment:
Have you tested this?
It looks suspicious that a connection **and** additional credentials were requested in the previous implementation. I am guessing this reflects the fact that the adapter needs a JDBC connection and the EXALoader uses the ORA connection.

If you only have one of them, you would need to derive the other one from it.

Reply:
I checked the old scripts with a virtual schema with IMPORT FROM ORA property and it doesn't have additional credentials as properties, so I thought that it's just a strange error message.

Reply:
I found the proof in the tests later anyway.